### ISSUE TITLE
Scheduler add-on

### ISSUE THREAD

**Issue by YugoAmaryl** [angular-ui/ui-calendar#317]
Is there any plans to make ui-calendar work with the new "scheduler" add-on?

**Comment by SamSlaught** [comment]
Yes, will this angular-ui calendar be able to work with the scheduler add on or not? I was planning on creating an Angular app with full calendar and the scheduler add on.

**Comment by joshkurz** [comment]
Thats a good question. I have not seen the scheduler yet. I don't see why
it would not work with it though. It's probably just some extra configs
needed to be passed through.

I dont have the time to check it out right now, if you want to do a POC
please go for it.

On Tue, Oct 20, 2015 at 9:42 AM, Sam Slaughterbeck <notifications@github.com
> wrote:

> Yes, will this angular-ui calendar be able to work with the scheduler add
> on or not? I was planning on creating an Angular app with full calendar and
> the scheduler add on.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/angular-ui/ui-calendar/issues/317#issuecomment-149572047>
> .
>



-- 
*Josh Kurz*
www.joshkurz.net

**Comment by gusreyes01** [comment]
I would also like this question answered.

**Comment by dwelch2344** [comment]
FWIW we have it working in a fairly large app, but the performance is AWEFUL. We've replicated the amount of data on the raw FC + Scheduler and it works fine, and profiling shows that the majority of our time is inside the digest loop so I'm guessing it's related to a event `$watch` somewhere

**Comment by ketsugi** [comment]
How did you get it working? I'm looking at the calendar.js source code and it's not at all obvious to me how to pass in the extra data.

**Comment by weslley39** [comment]
+1

**Comment by CodyMorris** [comment]
+1, happy to contribute to this also.  About to start building custom views for the scheduler add-on and would love to do so in angular instead of jquery

**Comment by benoit-nadaud** [comment]
Hi,

So is it working now ?  Did you fixed your performances problems @dwelch2344

**Comment by dwelch2344** [comment]
@ketsugi @benoit-nadaud Sorry for the lack of responses. We ended up circumventing angular and passing the data directly to the jQuery plugin. Then we just had to work around the nasty `$scope.$apply` workarounds for getting back into the digest. 

I've been on a new project for months so I haven't tried the latest

**Comment by alexreynier** [comment]
Hi, 

Is anyone working on this? I think it would be really useful to get this done, it'd avoid the jquery/angular talkback loops if used as standalone! I am happy to build the additional bits for it - but if there is already someone out there who has done all the hard work - it'd be silly to redo it!

Thanks!

**Comment by eripoll** [comment]
@dwelch2344 I would be very thankful if you could share your solution. It is vital for me to get this to work and I've wasted too much time already on this issue. Thanks in advance!

**Comment by kumarlalapet** [comment]
Is anyone working on this? Or have a solution for the same?

**Comment by smxdevst** [comment]
+1

**Comment by Babugoli** [comment]
Hi @dwelch2344 

Do you mind sharing How we can incorporate resource into uiCalender, I dont mind using apply.

  $scope.uiConfig = {
        calendar: {
            height: 520,
            editable: true,
            header: {
                left: 'title',
                center: '',
                right: 'timelineDay,timelineThreeDays,agendaWeek'
            },
            droppable: true,
            defaultView: 'timelineDay',
            theme: true,
            resourceLabelText: 'Team',
            //resources: [
			//	{ id: 'a', title: 'User 1' },
			//	{ id: 'b', title: 'User 2', eventColor: 'green' },
			//	{ id: 'c', title: 'User 3', eventColor: 'orange' },
			//	{ id: 'e', title: 'User 4' },
			//	{ id: 'f', title: 'User 5', eventColor: 'red' },
            //],
            events: [
				{ id: '1', resourceId: 'b', start: '2016-01-07T02:00:00', end: '2016-01-07T07:00:00', title: 'event 1' },
				{ id: '2', resourceId: 'c', start: '2016-01-07T05:00:00', end: '2016-01-07T22:00:00', title: 'event 2' },
				{ id: '3', resourceId: 'd', start: '2016-01-06', end: '2016-01-08', title: 'event 3' },
				{ id: '4', resourceId: 'e', start: '2016-01-07T03:00:00', end: '2016-01-07T08:00:00', title: 'event 4' },
				{ id: '5', resourceId: 'f', start: '2016-01-07T00:30:00', end: '2016-01-07T02:30:00', title: 'event 5' }
            ],
            //events: $scope.loadEvents,
            eventClick: $scope.alertOnEventClick,
            eventDrop: $scope.alertOnDrop,
            eventResize: $scope.alertOnResize,
            eventRender: $scope.eventRender,
            dayClick: $scope.dayClick,
           // drop: $scope.drop,

            //removeEvents:$scope.removeEvents,
        }
    };

1. How to I set resource to fullcalender. I cannot set it to uiCalender directive. 

Do you mind sharing example code snippet ? Appreciate it ?
